mimoto-issuers-config.json

```diff
@@ -1,311 +0,0 @@
-{
```

Comment:
Can we have a readme file which clearly calls out these files move to the inji-config repo and they can find it there ?

Reply:
@swatigoel to take care of this

Reply:
sure @vishwa-vyom, will create readme for this

Reply:
@dhanendra06 / @Varaniya201 Can you create a jira task for the same for tracking purpose, so we can mark this resolved ?

Reply:
Created task for @swatigoel  @vishwa-vyom

Reply:
https://mosip.atlassian.net/browse/MOSIP-35401